custom_components/cloudflare_tunnel_monitor/const.py

```diff
@@ -1,16 +1,13 @@
-DOMAIN = "cloudflare_tunnel_monitor"
-
-# Constantes para las claves de los datos de usuario
-CONF_EMAIL = "email"
-CONF_API_KEY = "api_key"
-CONF_ACCOUNT_ID = "account_id"
-
-# Constantes para las etiquetas que se mostrarán en la interfaz de usuario
-LABEL_EMAIL = "Email Address"
-LABEL_API_KEY = "API Key"
-LABEL_ACCOUNT_ID = "Account ID"
-
-# Constantes para los placeholders o valores sugeridos
-PLACEHOLDER_EMAIL = "example@example.com"
-PLACEHOLDER_API_KEY = "global-api-key"
-PLACEHOLDER_ACCOUNT_ID = "your-account-id"
+DOMAIN = "cloudflare_tunnel_monitor"
+
+# Constantes para las claves de los datos de usuario
+CONF_API_KEY = "api_key"
+CONF_ACCOUNT_ID = "account_id"
+
+# Constantes para las etiquetas que se mostrarán en la interfaz de usuario
+LABEL_API_KEY = "API Key"
+LABEL_ACCOUNT_ID = "Account ID"
+
+# Constantes para los placeholders o valores sugeridos
+PLACEHOLDER_API_KEY = "global-api-key"
```

Comment:
@syphernl This doesn't need to be "global-api-key" anymore :)

Reply:
Good catch! I thought I had removed/replaced all references to `global` but this one slipped through. Fixed 😄

Reply:
I've lost count the amount of times this has happened to me ;)